File: A/A1001.py
num = input().split()
a = -1 * int(num[0][1:]) if num[0][0] == "-" else int(num[0])
b = -1 * int(num[1][1:]) if num[1][0] == "-" else int(num[1])
ans = str(a+b)
final_ans = ""
while len(ans) > 3:
    final_ans = ","+ans[-3:]+final_ans
    ans = ans[:-3]
final_ans = ans + final_ans
if final_ans[0]=="-" and final_ans[1] == ",":
    final_ans = "-" + final_ans[2:]
print(final_ans)

File: A/A1001.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
  long a, b;
  cin>>a>>b;
  string ans = to_string(a+b);
  string final_ans = "";
  int length = ans.length();
  for(int i = 1;i<=length;i++){
    final_ans = ans[length-i]+final_ans;
    if(i%3 == 0 && i !=length){
      final_ans = ","+final_ans;
    }
  }
  cout<<final_ans;
  return 0;
}